plugin/lsp.vim

```diff
@@ -21,7 +21,10 @@ let g:lsp_signs_information = get(g:, 'lsp_signs_information', {})
 let g:lsp_signs_hint = get(g:, 'lsp_signs_hint', {})
 let g:lsp_signs_priority = get(g:, 'lsp_signs_priority', 10)
 let g:lsp_signs_priority_map = get(g:, 'lsp_signs_priority_map', {})
+let g:lsp_documentation_debounce = get(g:, 'lsp_documentation_debounce', 0)
```

Comment:
It should be 100 by default.

I think this debouncing has no bad effect so we should enable debouncing by default.

I checked the current debouncing behaviors that worked well.

Reply:
OK

Reply:
I think it should be kept disabled by default (i.e. value should be 0)... I
 want to hear what other say.

<br>

@mattn @prabirshrestha @thomasfaingnaert What do you think? What should be the default value?

Reply:
for perf it is better to debounce primarily since vim script is very slow.

Reply:
OK.. so debounce by default.

Reply:
i dont have a strong opinion on this. can we be easily changed later as long as it is document on what it does and what values it should be wouldnt matter.

Reply:
OK. So I am setting the default to 80ms as suggested by @hrsh7th